components/data-table/index.jsx

```diff
@@ -517,6 +513,16 @@ class DataTable extends React.Component {
 				<div
 					className="slds-table_header-fixed_container"
 					style={{ height: '100%' }}
+					onScroll={(e) => {
```

Comment:
Can we add more conditions to this? Maybe something like if horizontal scrolling can occur (that is something like measuring `clientWidth`?).